Source/WebCore/platform/graphics/Font.h

```diff
@@ -378,6 +385,8 @@ class Font : public RefCounted<Font>, public CanMakeSingleThreadWeakPtr<Font> {
     
     unsigned m_allowsAntialiasing : 1;
 
+    IsSystemFallbackFontPlaceholder m_isSystemFontFallbackPlaceholder { IsSystemFallbackFontPlaceholder::No };
```

Comment:
Oh, I think we wanna use unsigned & bitfield of 1. Otherwise, it's gonna add paddings.